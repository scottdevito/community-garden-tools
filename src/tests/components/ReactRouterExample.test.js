import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow, mount, render } from 'enzyme';
import { BrowserRouter as Router } from 'react-router-dom';

import ReactRouterExample from '../../components/ReactRouterExample';

// Snapshot test
it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <ReactRouterExample />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
