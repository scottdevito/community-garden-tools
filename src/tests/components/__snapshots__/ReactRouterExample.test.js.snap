// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bZQynM cZVOIA"
>
  <h1>
    React Router
  </h1>
  <div
    className="sc-ifAKCX sPqpe"
  >
    <h3
      className="sc-EHOje ffdZoo"
    >
      Resources:
    </h3>
    <a
      href="https://reacttraining.com/react-router/web/guides/quick-start"
    >
      https://reacttraining.com/react-router/web/guides/quick-start
    </a>
  </div>
  <h4>
    Test routing:
  </h4>
  <a
    href="/react-router-test"
    onClick={[Function]}
  >
    <button
      className="sc-bxivhb iBHaDI"
    >
      Some Link
    </button>
  </a>
</div>
`;
